AK/SIMD.h

```diff
@@ -106,11 +106,19 @@ struct IndexVectorFor<T> {
         u64 __attribute__((vector_size(sizeof(T))))>;
 };
 
+template<typename T, size_t element_count>
+struct MakeVectorImpl {
+    using Type __attribute__((vector_size(sizeof(T) * element_count))) = T;
+};
```

Comment:
You sure this works on GCC?
I had issues with dependent vector sizes on there

Reply:
Yes, this works on my machine, Ubuntu with GCC. Source: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=88600#c1